src/main/java/cinema/mappers/OrderMapper.java

```diff
@@ -0,0 +1,42 @@
+package cinema.mappers;
+
+import cinema.model.Order;
+import cinema.model.Ticket;
+import cinema.model.User;
+import cinema.model.dto.OrderRequestDto;
+import cinema.model.dto.OrderResponseDto;
+import cinema.service.ShoppingCartService;
+import cinema.service.UserService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderMapper {
+    public List<Ticket> mapFromOrderRequestDtoToTicketList(
+            OrderRequestDto orderRequestDto, UserService userService,
+            ShoppingCartService shoppingCartService) {
+        return shoppingCartService.getByUser(userService
+                .findById(orderRequestDto.getUserId())).getTickets();
+    }
+
+    public User mapFromOrderRequestDtoToUser(OrderRequestDto orderRequestDto,
+                                             UserService userService) {
+        return userService.findById(orderRequestDto.getUserId());
+    }
```

Comment:
we won't need these methods when we refactor the code

Reply:
done